Library/Formula/testssl.rb

```diff
@@ -0,0 +1,13 @@
+class Testssl < Formula
+  desc "Tool which checks for the support of TLS/SSL ciphers and flaws"
+  homepage "https://testssl.sh/"
+  url "https://github.com/drwetter/testssl.sh/archive/v2.6.tar.gz"
+  version "2.6"
+  sha256 "286b3285f096a5d249de1507eee88b14848514696bc5bbc4faceffa46b563ebd"
+
+  depends_on "openssl"
+
+  def install
+    bin.install "testssl.sh"
+  end
+end
```

Comment:
Thanks for ideas. `testssl.sh` is not a library so that I could test if it's present and usable by compiling some code. It's normally used as an online test.

But what about running `testssl.sh --local`? That seems like a reasonable way to test things are working.